tests/test_meshtastic_utils_reconnect.py

```
@@ -0,0 +1,174 @@
+
+
+@pytest.mark.usefixtures("reset_meshtastic_globals")
+@pytest.mark.asyncio
+class TestReconnectSuccess:
+    async def test_reconnect_succeeds_and_clears_future_and_flag(self):
+        mock_client = MagicMock()
+        mu.reconnecting = True
+        mu.shutting_down = False
+        mu.reconnect_task_future = None
+
+        def _connect_side_effect(_cfg, _force):
```

Ruff (0.15.9)

[warning] Missing return type annotation for private function `_connect_side_effect`

(ANN202)